policyengine_canada/variables/gov/cra/tax/income/credits/climate_action/climate_action_incentive_single_parent.py

```diff
@@ -0,0 +1,25 @@
+from policyengine_canada.model_api import *
+
+
+class climate_action_incentive_single_parent(Variable):
+    value_type = float
+    entity = Household
+    label = "Canada Climate Action single parent amount"
+    unit = CAD
+    documentation = "Determination wether the filer is a single parent eligible for the climate action incentive"
+    definition_period = YEAR
+
+    def formula(household, period, parameters):
+        single_parent = household("is_single_parent", period)
+        province = household("province_str", period)
+        first_child_amount = parameters(
+            period
+        ).gov.cra.tax.income.credits.climate_action_incentive.amount.first_child_in_single_parent_family[
+            province
+        ]
+        children = household("climate_action_incentive_children", period)
+        return where(
+            single_parent,
+            children + (first_child_amount / 2),
```

Comment:
Unless `/2` is part of the legal code, let's stick to what the document says: single parents get a bonus equal to the first-child amount minus the per-child amount.